oof/src/lib.rs

```diff
@@ -239,6 +239,19 @@ mod tests {
         assert!(!flags.is_present("output_file"));
     }
 
+    #[test]
+    fn test_unknown_short_flag() {
+        let flags_info = [
+            ArgFlag::long("output_file").short('o'),
+            Flag::long("verbose").short('v'),
+            Flag::long("help").short('h'),
+        ];
+
+        let args = gen_args("ouch a.zip -s b.tar.gz");
+        let result = filter_flags(args, &flags_info).unwrap_err();
+        assert!(matches!(result, OofError::UnknownShortFlag('s')));
```

Comment:
This procedure could be inserted inside of a closure or function (Rust supports functions inside of functions), so that you could test with other strings other `"ouch a.zip -s b.tar.gz"`, using the same `flags_info`, but with different args input that should return the same `OofError::UnknownShortFlag('s')`, that using multiple `assert!`s.

Reply:
Got it, will do